### PR TITLE
Azure Analysis Services: added new sku and scale up/down support

### PR DESCRIPTION
**Pull request by igor-uzhviev**
## Description
Added new SKUs for Azure Analysis Services, added support of scale down/up.

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md).**
- [x] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md).**
- [x] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.
- [x] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [x] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change).
- [x] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [x] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [x] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [x] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1 [hunk 1]
```diff
@@ -123,6 +123,64 @@ function Test-AnalysisServicesServer
...
 
+<#
+.SYNOPSIS
+Tests scale up and down of Analysis Services server (B1 -> S2 -> S1).
+#>
+function Test-AnalysisServicesServerScaleUpDown
+{
+    param
+	(
+		$location = "West US"
```
Comment (markcowl) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1]: You shoudls et the location based on the availability of the RP in the location and not to a hard-code value.  Ths prevents using this test against the test cluster, sovereign clouds, etc.

Comment (igor-uzhviev) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1]: I updated the change log.
Sent you an email with questions about location. It is not really clear what and how should be implemented.

Comment (igor-uzhviev) [inline src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Test/ScenarioTests/AsTests.ps1]: Added GetLocation as discussed by email.

File: src/ResourceManager/AnalysisServices/ChangeLog.md [hunk 2]
```diff
@@ -19,11 +19,9 @@
 -->
 ## Current Release
 
-## Version 0.3.0
-
-## Version 0.2.0
-
-## Version 0.1.0
+## Version 0.4.0
```
Comment (markcowl) [inline src/ResourceManager/AnalysisServices/ChangeLog.md]: Please just add this under 'Current release and leave the version number as is

**Comment by markcowl** [comment]
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1606/

**Comment by markcowl** [comment]
@igor-uzhviev note: if yopu just change the changelog, the on demand run will not need to be rerun

**Comment by igor-uzhviev** [comment]
change log is fixed. please verify.

**Pull request merged**